CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.17.0 (date-to-be-updated-on-release)
+* [knife-ec2:#516](https://github.com/chef/knife-ec2/pull/516) Fix the review comments for the PR #514.
+* [knife-ec2:#515](https://github.com/chef/knife-ec2/pull/515) Add primary eni to server.
```

Comment:
The description of PR can be better to understand whats added and edit the same here